### DISCUSSION TITLE
Looking for clarification from the motion correction output

### DISCUSSION THREAD

**Discussion by zhouyi0812** [flatironinstitute/CaImAn#1258] [Q&A]
Hello! 
I am a bit confused that after motion correction I have two output as follow:
1) 002C4d7_B2_B5_1_00d00h00m_rig__d1_1152_d2_1536_d3_1_order_F_frames_540.mmap
2) cnmf-memmap_d1_1152_d2_1536_d3_1_order_C_frames_540.mmap
which one is the correct one that we can use for the following source extraction? 
Thank you!

**Comment by EricThomson** [comment]
This is something that we just added a discussion for in the most recent release: 
https://caiman.readthedocs.io/en/latest/Getting_Started.html#memory-mapping

The C-order one is the one used by CNMF.  F-order is optimized for motion correction.

**Comment by zhouyi0812** [comment]
So after motion correction, I can reload the F order mmap to continue with source extraction if I am using cnmfe?
Thank you!

**Reply by EricThomson** [reply]
I would recommend follow the demo. See `fname_mc` (which is F order). Then we resave as `fname_new` which is C order, which is optimized for CNMF, and what is used for subsequent processing.  This is in `demo_pipeline_cnmfE.ipynb`. I recommend print out those filenames in the notebook and be sure to follow the logic, and the discussion of memory mapping in the docs I just linked to. It can be confusing, and ideally it would be nice to hide all this back-end detail. 

I assume you ask because you want to know what you can safely delete, though. In that case, you can delete `fname_mc` as the file referred to in  `fname_new` is what you want. That is what is linked to in `cnmfe_model.mmap_file` for instance, and used for cnmf.

**Reply by EricThomson** [reply]
The new notebooks are up with much more explanation (for instance of `corr/pnr`).
